components/developer/automake-1.16/Makefile

```diff
@@ -21,31 +21,30 @@
 
 #
 # Copyright (c) 2012, Oracle and/or its affiliates. All rights reserved.
-# Modified by Aurelien Larcher, 2015-2018.
+# Modified by Aurelien Larcher, 2015-2020.
 #
 
 include ../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME=		automake
 COMPONENT_MAJOR_VERSION=1.16
-COMPONENT_VERSION=	$(COMPONENT_MAJOR_VERSION).1
+COMPONENT_VERSION=	$(COMPONENT_MAJOR_VERSION).2
 COMPONENT_FMRI=		developer/build/automake-116
 COMPONENT_CLASSIFICATION=Development/GNU
 COMPONENT_SRC=		$(COMPONENT_NAME)-$(COMPONENT_VERSION)
 COMPONENT_ARCHIVE=	$(COMPONENT_SRC).tar.gz
 COMPONENT_ARCHIVE_HASH=	\
-  sha256:608a97523f97db32f1f5d5615c98ca69326ced2054c9f82e65bade7fc4c9dea8
+  sha256:b2f361094b410b4acbf4efba7337bdb786335ca09eb2518635a09fb7319ca5c1
 COMPONENT_ARCHIVE_URL=	http://ftp.gnu.org/pub/gnu/automake/$(COMPONENT_ARCHIVE)
 COMPONENT_SUMMARY=	A Makefile generator
 COMPONENT_PROJECT_URL=  http://www.gnu.org/software/automake/ 
-COMPONENT_LICENSE_FILE=	$(COMPONENT_NAME).license
 COMPONENT_LICENSE=	GPLv2 FDLv1.3
 
 include $(WS_MAKE_RULES)/common.mk
 
 PKG_MACROS+= COMPONENT_MAJOR_VERSION=$(COMPONENT_MAJOR_VERSION)
 
-CONFIGURE_OPTIONS  +=	--program-transform-name=s/$$/-$(COMPONENT_MAJOR_VERSION)/
+#CONFIGURE_OPTIONS  +=	--program-transform-name=s/$$/-$(COMPONENT_MAJOR_VERSION)/
```

Comment:
Did this option disappear?

Reply:
The behaviour changed since 1.16.1 and generates a different name aclocal-1.16-1.16.1 (for 1.16.2!)... so I just kept the default name and changed the manifest as:
`file usr/bin/aclocal path=usr/bin/aclocal-$(COMPONENT_MAJOR_VERSION)`